ui-tests/src/test/java/org/openmrs/reference/page/RequestAppointmentPage.java

```diff
@@ -21,19 +28,36 @@ public void enterAppointmentType(String type) {
         clickOn(SERVICE_DROPDOWN);
     }
 
-    public void enterValue(String value) {
-        setText(FRAME_VALUE, value);
+    public void enterMinimumValue(String value) {
+        setText(MIN_FRAME_VALUE, value);
     }
 
-    public void selectUnits(String units) {
-        selectFrom(FRAME_UNITS, units);
+    public void selectMinimumUnits(String units) {
+        selectFrom(MIN_FRAME_UNITS, units);
     }
 
+    public void enterMaximumValue(String value) {
+        setText(MAX_FRAME_VALUE, value);
+    }
+    
+    public void selectMaximumUnits(String units) {
+        selectFrom(MAX_FRAME_UNITS, units);
+    }
+    
+    public void addAppointmentRequestNote(String note) {
+        findElement(APPOINTMENT_REQUEST_NOTE).clear();
+        setText(APPOINTMENT_REQUEST_NOTE, note); 	
```

Comment:
setText should automatically overwrite the existing value

Reply:
> setText should automatically overwrite the existing value

Does this mean eliminating findElement().clear(); completely? and every where I had applied it as well